WebHookHandlers/Telegram/Actions/IAction.cs

```diff
@@ -0,0 +1,6 @@
+namespace JewishBot.WebHookHandlers.Telegram.Actions
+{
+    internal interface IAction
```

Comment:
I am in search for good interface implementation. Maybe I should use last argument as params array or smth, because in some cases I need more params